src/library/scala/collection/ArrayOps.scala

```diff
@@ -559,7 +559,7 @@ final class ArrayOps[A](private val xs: Array[A]) extends AnyVal {
 
   /** Selects all elements of this array which do not satisfy a predicate.
     *
-    *  @param pred  the predicate used to test elements.
+    *  @param p     the predicate used to test elements.
     *  @return      a new array consisting of all elements of this array that do not satisfy the given predicate `pred`.
```

Comment:
The return entry mentions `pred`, so change to `p`?

Reply:
Thanks! Will fix it.